docs/operation/SOP/PM.DWD.3/PM.DWD.3-UpdatingPipelineParameters.md

```diff
@@ -0,0 +1,107 @@
+# Updating Pipeline Parameters
+
+From time-to-time there may be a requirement to add or subtract pipeline parameters.
+
+Unfortunately, the DRAFT payload inputs we supply do not directly map to the pipeline parameters used when running ICAv2.
+This is a little annoying but also improves the user experience as is allows us to provide more user-friendly names for input parameters.
+This assumes that the [CWL pipeline][cwl-pipeline] has already been updated to support the new parameters. See [New Dragen WGTS DNA Deployment SOP][new_dragen_wgts_dna_deployment_sop] for more information.
+
+## Constants File Update
+
+To update any of our pipeline parameters, head to the infrastructure stage constants path and you should find something like the following.
```

Comment:
Add path/link?
e.g. "... head to [infrastructure/stage/constants.ts](../../../../infrastructure/stage/constants.ts) ..."